docs/user/process.rst

```diff
@@ -79,6 +117,285 @@ outputWidth   Width of the output raster in pixels.
 outputHeight  Height of the output raster in pixels.
 ============  ===========
 
+.. _date_offset_process:
+
+DateOffsetProcess
+^^^^^^^^^^^^^^
+
+The ``DateOffsetProcess`` modifies the specified date field in a feature collection by an input time period.
+
+============  ===========
+Parameters    Description
+============  ===========
+data          Input features.
+dateField     The date attribute to modify.
+timeOffset    Time offset (e.g. P1D).
+============  ===========
+
+
+
+.. _hash_process:
+
+HashAttributeProcess
+^^^^^^^^^^^^^^^^^^^^
+
+The ``HashAttributeProcess`` adds an attribute to each SimpleFeature that hashes the configured attribute modulo the configured param.
+
+============  ===========
+Parameters    Description
+============  ===========
+data          Input Simple Feature Collection to run the hash process over.
+attribute     The attribute to hash on.
+modulo        The divisor.
+============  ===========
+
+.. _hashExampleXML:
+
+Hash example (XML)
+"""""""""""""""""""
+
+:download:`HashAttributeProcess_wps.xml </user/_static/process/HashProcess_wps.xml>` is a geoserver WPS call to the GeoMesa HashAttributeProcess. It can be run with the following curl call:
+
+.. code-block:: bash
+
+    curl -v -u admin:geoserver -H "Content-Type: text/xml" -d@HashAttributeProcess_wps.xml localhost:8080/geoserver/wps
+
+.. _queryExampleResults:
```

Comment:
this is a duplicate tag - you might not need an anchor here if you're not referencing it anywhere